schemas/vertices/publications.json

```diff
@@ -3,8 +3,20 @@
   "type": "object",
   "additionalProperties": false,
   "description": "The citation for an academic publication",
```

Comment:
Glad we are split this into its own vertex. We also want to ref this from catalog apps